es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -132,13 +132,13 @@
 
 **23. Vocabulary ― When selecting a model, we distinguish 3 different parts of the data that we have as follows:**
 
-&#10230; Vocabulario ― Cuando se selección un modelo, se tienen 3 diferentes partes de la información como se muestra:
+&#10230; Vocabulario ― Cuándo se selección un modelo, se tienen 3 diferentes partes de la información como se muestra:
```

Comment:
In this case I consider "cuando" is used as adverb but not interrogative. If "cuándo" can be replaced by "en qué momento," it must be stressed. Also, "selección" should be "selecciona".

(https://www.fundeu.es/recomendacion/como-cuando-y-donde-cuando-se-escriben-con-tilde-509/)

Reply:
You're completely right. My bad. Also, is _selecciona_ not _selección_.